.github/workflows/ci.yml

```diff
@@ -45,24 +58,19 @@ jobs:
         uses: cypress-io/github-action@8aac1d019734a107e4eaaefe2e26beb3149e5540
         with:
           spec: cypress/integration/index.spec.js
-      - name: Commit
-        if: ${{ github.event_name == 'push' }}
+      - name: Get Protected Status
+        if: github.event_name == 'push'
+        id: protected_step
         run: |
-          git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
-          git commit package.json yarn.lock -m "Updating package.json and yarn.lock" || echo "Nothing to commit"
-      - name: Get Branch Name
-        if: ${{ github.event_name == 'push' }}
-        id: branch_name
+          PROTECTED=$(curl "https://api.github.com/repos/bradlhart/eosjs/branches/${GITHUB_REF#refs/*/}" 2>/dev/null | jq -r '.protected')
```

Comment:
curling your personal repo here. The $ORG part of the URL should be deduced if possible, and if not hardcoded to EOSIO

Reply:
Switched it with `${{ github.repository }}` which equals "EOSIO/eosjs"